external/configuration-guides/vpn.md

```diff
@@ -213,15 +213,22 @@ It has been tested with Fedora 23 and Debian 8 templates.
    ;;
    esac
    ~~~
-
+   You can add in above script following if you want (it is not necessary)
+   
+   vpn_dns="X.X.X.X"
+   
+   addr="X.X.X.X"
+   
+   Replace X.X.X.X with DNS Addresses provided by your VPN (like 10.8.0.1 in Mullvad,10.0.254.2 in IVPN,103.86.99.100 in NordVPN etc. We do not endorse any particular VPN. These are just example to understand the concept)
```

Comment:
VPN servers provide this information automatically and it is bad idea to override it - it can easily lead to broken configuration if you (or VPN provider) change configuration.

Reply:
What do you really want it here? Should it be removed completely or just to remove the part within brackets?

Reply:
Remove completely

Reply:
Although I have removed this part but having this part as it was in commit da2511f may have beneficial effect for end users who want extra benefits from their VPN. As many VPNs now a days are offering special features on different DNS addresses which don't depend on servers location.